<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QXHexSearch</class>
 <widget class="QDialog" name="QXHexSearch">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>285</width>
    <height>304</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>285</width>
    <height>304</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>285</width>
    <height>304</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Search</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelSearchFrom">
       <property name="text">
        <string>Search from:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBoxX" name="comboBoxSearchFrom">
       <item>
        <property name="text">
         <string>Begin</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cursor</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidgetX" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabSignature">
      <attribute name="title">
       <string>Signature</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QTextEditXHEX" name="textEditData"/>
       </item>
       <item>
        <widget class="QLabel" name="labelHEXInfo">
         <property name="text">
          <string>&quot;A-F&quot;, &quot;0-9&quot; ,&quot;?&quot;,&quot;.&quot; and Space allowed</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelHEXInfo2">
         <property name="text">
          <string>&quot;?&quot; and &quot;.&quot; as Wildcards</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Max 256 bytes</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabText">
      <attribute name="title">
       <string>Text</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QTextEdit" name="textEditText"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="labelTextInfo">
             <property name="text">
              <string>Max 256 symbols</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QCheckBoxX" name="checkBoxMatchCase">
               <property name="text">
                <string>Match case</string>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QRadioButtonX" name="radioButtonANSI">
             <property name="text">
              <string notr="true">ANSI</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButtonX" name="radioButtonUnicode">
             <property name="text">
              <string notr="true">Unicode</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabValues">
      <attribute name="title">
       <string>Value</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEditX" name="lineEditValue">
           <property name="text">
            <string notr="true">0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEditX" name="lineEditHEX">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxEndianness">
         <property name="title">
          <string>Endianness</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QRadioButtonX" name="radioButtonBigEndian">
            <property name="text">
             <string notr="true">Big-Endian</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QRadioButtonX" name="radioButtonLittleEndian">
            <property name="text">
             <string notr="true">Little-Endian</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxType">
         <property name="title">
          <string>Type</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QRadioButtonX" name="radioButtonChar">
              <property name="text">
               <string notr="true">char</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QRadioButtonX" name="radioButtonShort">
              <property name="text">
               <string notr="true">short</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QRadioButtonX" name="radioButtonInt">
              <property name="text">
               <string notr="true">int</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QRadioButtonX" name="radioButtonInt64">
              <property name="text">
               <string notr="true">int64</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QRadioButtonX" name="radioButtonUChar">
              <property name="text">
               <string notr="true">uchar</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QRadioButtonX" name="radioButtonUShort">
              <property name="text">
               <string notr="true">ushort</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QRadioButtonX" name="radioButtonUInt">
              <property name="text">
               <string notr="true">uint</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QRadioButtonX" name="radioButtonUInt64">
              <property name="text">
               <string notr="true">uint64</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QRadioButtonX" name="radioButtonDouble">
              <property name="text">
               <string notr="true">double</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QRadioButtonX" name="radioButtonFloat">
              <property name="text">
               <string notr="true">float</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButtonX" name="pushButtonOK">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButtonX" name="pushButtonCancel">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QPushButtonX</class>
   <extends>QPushButton</extends>
   <header>qpushbuttonx.h</header>
  </customwidget>
  <customwidget>
   <class>QLineEditX</class>
   <extends>QLineEdit</extends>
   <header>qlineeditx.h</header>
  </customwidget>
  <customwidget>
   <class>QTabWidgetX</class>
   <extends>QTabWidget</extends>
   <header>qtabwidgetx.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QCheckBoxX</class>
   <extends>QCheckBox</extends>
   <header>qcheckboxx.h</header>
  </customwidget>
  <customwidget>
   <class>QComboBoxX</class>
   <extends>QComboBox</extends>
   <header>qcomboboxx.h</header>
  </customwidget>
  <customwidget>
   <class>QRadioButtonX</class>
   <extends>QRadioButton</extends>
   <header>qradiobuttonx.h</header>
  </customwidget>
  <customwidget>
   <class>QTextEditXHEX</class>
   <extends>QTextEdit</extends>
   <header>qtexteditxhex.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
